e/php/bootstrap

```diff
@@ -16,42 +16,61 @@ $lambdaRuntimeApi = getenv('AWS_LAMBDA_RUNTIME_API');
 require __DIR__ . '/vendor/autoload.php';
 
 $socket = '/tmp/php-fpm.sock';
-$config =  __DIR__ .'/php-fpm.conf';
-$cgi = new Process("php-fpm --fpm-config {$config} -c /opt/php.ini -d extension_dir=/opt/php/extensions");
-$cgi->setTimeout(null);
-$cgi->start(function ($type, $output) {
-    if ($type === Process::ERR) {
-        echo($output);
-        exit(1);
-    }
-});
-
-
-register_shutdown_function(function() use(&$cgi) {
-    $cgi->stop();
-});
-
-$wait = 5000; // 5ms
-$timeout = 5000000; // 5 secs
-$elapsed = 0;
-while (! file_exists($socket)) {
-    usleep($wait);
-    clearstatcache();
-    $elapsed += $wait;
-    if ($elapsed > $timeout) {
-        echo "Timeout while waiting for socket at {$socket}..";
-        exit(1);
+
+function is_server_started($socket)
+{
+    return file_exists($socket);
+}
+
+function wait_for_server($socket)
+{
+    $wait = 5000; // 5ms
+    $timeout = 5000000; // 5 secs
+    $elapsed = 0;
+    while (! is_server_started($socket)) {
+        usleep($wait);
+        clearstatcache();
```

Comment:
You should pass in the socket to only clear this single entry

Reply:
What do you mean, there is only one entry?

Reply:
I guess the symfony process class internally also uses some descriptors whicha cache you might invalidate with this.

To act as local as possible, I propose to pass the argt to clearstatcache

Reply:
Actually, it's the second param. See http://php.net/manual/en/function.clearstatcache.php